### DISCUSSION TITLE
Logs to be visible from the App

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#6296] [Feature Requests]
# Logs to visible from the App

## Idea

View all logs that are currently written to `stdout` using Pino from the App

The conversation started on Discord with @rijkvanzanten about how could work, and so far:

- Streamline the logs using subscriptions with WebSockets, to stream those log lines to the client-side for authenticated admin users
- Pino's existing hooks to write to a buffer to store the last X log lines
- Using Redis to rely on 'volatile' logs
- Implement similar as Drupal with a watchdog table, and perhaps create a Logger method that can be used from the extensions

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
